src/core/chart-generators.js

```diff
@@ -425,13 +427,23 @@ function shortestUniqueTimeseriesNamingFunction(metadata, data) {
   if (variation.length === 0) {
     throw new Error('Error: cannot graph identical timeseries');
   }
+  
+  // Build a dictionary with the base name for each variable (typically
+  // either "Mean" or "Mean Count") from the variable config file. 
+  // Defaults to "Mean" since all data displayed by this graph is MYMs.
+  const variables = _.uniq(_.pluck(metadata, "variable_id"));
```

Comment:
Underscore `_.pluck` is replaced in Lodash by `_.map` (special case of a string iteratee). One or the other of us will have to make this change, depending on in what order we merge PRs (mine is #317). How do you want to do this?